runners/core-construction-java/src/main/java/org/apache/beam/runners/core/construction/RehydratedComponents.java

```diff
@@ -71,8 +71,14 @@
               new CacheLoader<String, Coder<?>>() {
                 @Override
                 public Coder<?> load(String id) throws Exception {
-                  return CoderTranslation.fromProto(
-                      components.getCodersOrThrow(id), RehydratedComponents.this);
+                  RunnerApi.Coder coder;
+                  try {
+                    coder = components.getCodersOrThrow(id);
```

Comment:
Is it worth considering the `getOrDefault` style here? e.g.
```
getCodersOrDefault(id, null);
checkState(coder != null);
```

(If we ever put a null value in we're super broken anyway, right?)

Reply:
That's nice. Cleans the code, trims the stack trace.